src/GraphQL.NewtonsoftJson/OperationMessageJsonConverter.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using GraphQL.Transport;
+using Newtonsoft.Json;
+
+namespace GraphQL.NewtonsoftJson
+{
+    /// <summary>
+    /// A custom JsonConverter for reading or writing a <see cref="OperationMessage"/> object.
+    /// </summary>
+    public class OperationMessageJsonConverter : JsonConverter
+    {
+        private const string TYPE_KEY = "type";
+        private const string ID_KEY = "id";
+        private const string PAYLOAD_KEY = "payload";
+
+        /// <inheritdoc/>
+        public override bool CanConvert(Type objectType) => objectType == typeof(OperationMessage);
```

Comment:
https://github.com/graphql-dotnet/graphql-dotnet/pull/16

Worth to think about it for all new converters.

Reply:
I have thought about it, for `GraphQLRequest` and `OperationMessage` anyway.  If someone needs a derived class, they probably want to serialize/deserialize additional fields, and so the existing converter is insufficient.  Also, they can use `JsonMember` or whatever the serializer-specific attribute is to name the fields; they probably don't need to write a custom `JsonConverter`.  So it seemed best to match the exact type.

Reply:
ok